daget/__main__.py

```diff
@@ -22,22 +22,22 @@ def main():
   # get doi/url and resolve to landing page
   try:
     url = get_redirect_url(args.url)
+    res = [ele for ele in ["dataverse.harvard.edu", "dataverse.no", "snd.se/catalogue", "snd.gu.se", "su.figshare.com", "figshare.scilifelab.se", "zenodo.org"] if(ele in url)]
+    if res is False:
+      raise RepoError(f'Landing page is not supported {url}')
   except ResolveError as err:
-    print(bcolors.FAIL, f'error resolving {args.url}', bcolors.ENDC)
+    print(bcolors.FAIL, f'Error resolving {args.url}: {err}', bcolors.ENDC)
+    exit(1)
+  except Exception as e:
+    # Catch any other unexpected exceptions, including URLError
+    print(bcolors.FAIL, f'{e}', bcolors.ENDC)
     exit(1)
   
   print(f'landing page: {url}')
 
   # get desitnation directory and create directory
   desitnation = os.path.realpath(args.destination)
 
-  if not os.path.exists(desitnation):
```

Comment:
not sure why this was removed, checking for empty destination directory was a feature i added to make sure the downloaded dataset will be exactly as the remote source.